website/app-templates/smarty/head.tpl

```diff
@@ -2,7 +2,15 @@
     <meta charset="utf-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <title>GeoKrety: {block name=title}{/block}</title>
-    <meta name="description" content="" />
+    <meta name="title" content="GeoKrety"/>
+    <meta name="description" content="Open source item tracking for all geocaching platforms"/>
+    <meta property="og:type" content="website"/>
+    <meta property="og:url" content="https://geokrety.org/"/>
+    <meta property="og:title" content="GeoKrety"/>
+    <meta property="og:description" content="Open source item tracking for all geocaching platforms"/>
+    <meta property="twitter:url" content="https://geokrety.org/"/>
```

Comment:
We have a twitter account, should it goes there?
https://twitter.com/geokrety

Reply:
This meta field is for the URL of the website.